### PR TITLE
increase PE # for 35day run

### PR DESCRIPTION
**Pull request by jiandewang**
# PR Checklist

- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Provide a detailed description of what this PR does. What bug does it fix, or what feature does it add? Is a change of answers expected from this PR? Are any library updates included in this PR (modulefiles etc.)?

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [ ] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

**Comment by jiandewang** [comment]
@junwang-noaa can you merge this ? Current PE setting will not be able to finish 35 day run, the modified PE numbers are identical to what were used in workflow P6.

File: tests/tests/cpld_bmarkfrac_wave_v16_35d [hunk 1]
```diff
@@ -55,6 +55,16 @@ export ocn_petlist_bounds="600 839"
 export ice_petlist_bounds="840 887"
 export wav_petlist_bounds="888 1009"
 
+# 12x16 setting
+export TASKS=1700
+export INPES=12
```
Comment (junwang-noaa) [inline tests/tests/cpld_bmarkfrac_wave_v16_35d]: Please change to: INPES=16 and JNEPS=12

File: tests/tests/cpld_bmarkfrac_wave_v16_noahmp_35d [hunk 2]
```diff
@@ -46,14 +46,24 @@ export WRTTASK_PER_GROUP=24
...
-export TASKS=1010
-export INPES=8
-export JNPES=12
-export med_petlist_bounds="0 575"
-export atm_petlist_bounds="0 599"
-export ocn_petlist_bounds="600 839"
-export ice_petlist_bounds="840 887"
-export wav_petlist_bounds="888 1009"
+#export TASKS=1010
+#export INPES=8
```
Comment (junwang-noaa) [inline tests/tests/cpld_bmarkfrac_wave_v16_noahmp_35d]: Same here, INPES=12, JNEPS=8

Comment (jiandewang) [reply tests/tests/cpld_bmarkfrac_wave_v16_noahmp_35d]: corrected.

File: tests/tests/cpld_bmarkfrac_wave_v16_35d (hunk 1 above)
Comment (jiandewang) [reply tests/tests/cpld_bmarkfrac_wave_v16_35d]: corrected

**Review by junwang-noaa — APPROVED** [review]

**Pull request merged**